server/src/infra/database.config.ts

```diff
@@ -20,6 +20,7 @@ export const databaseConfig: PostgresConnectionOptions = {
   subscribers: [__dirname + '/subscribers/*.{js,ts}'],
   migrationsRun: false,
   connectTimeoutMS: 10000, // 10 seconds
+  parseInt8: true,
```

Comment:
This should be a separate PR imo.

Reply:
I require the change for this PR, unless I want to change the type back to string. We only have one usage of bigint and that's all this change should affect.

Reply:
That's fine. I think we have more than one use of this though and it is just easier to test as a separate change. This returns all _numbers_ as numbers instead of strings, right?

Reply:
My understanding is it parses all 8 byte number types into a number in typeorm. Postgres number types are variable length and to my understanding will still be strings. This also only affects integer types, so double precision won't be affects either. It should just be bigint and bigserial.